scripts/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# Set current project root
+PROJ_ROOT=$(pwd)
+
+# Clean out patches
+printf "\n======== Cleaning out patches ========\n"
+quilt pop -a
+
+# make sure module is current
+printf "\n======== Updating submodule ========\n"
+git submodule update --init
+
+# Apply patches
+printf "\n======== Applying patches ========\n"
+{
+    quilt push -a --leave-rejects --color=auto 
```

Comment:
what does `--leave-rejects` do ?

Reply:
this will create a file when there are failed hunks on a patch. so for example if part of the patch on product.json fails it will create a file product.json.rej - for the reject parts. If there is a failure line 20 prints a message, line 21 shows the *.rej files and then it will exit the script